_includes/toc.html

```diff
@@ -19,6 +19,7 @@
             * h_max      (int)    :   6    - the maximum TOC header level to use; any header greater than this value will be ignored
             * ordered    (bool)   : false  - when set to true, an ordered list will be outputted instead of an unordered list
             * item_class (string) :   ''   - add custom class for each list item; has support for '%level%' placeholder, which is the current heading level
+            * base_url    (string) :   ''   - Possible to add links to other domains
```

Comment:
For consistency, could you remove the extra space before the `(string)` so everything can be lined up and make `Possible` lowercase to match the others?

Additionally, could you also add this new parameter to the README?